packages/notifier/README.md

```diff
@@ -94,55 +102,54 @@ observeIteration(subscription, observer);
 // finished done
 ```
 
-The difference between the two pairs is that the NotifierKit is a lossy
-conveyor of non-final values whereas the SubscriptionKit is a lossless conveyor
-of non-final values. They both losslessly convey termination, so the
-SubscriptionKit losslessly conveys the entire async iteration.
+Note that SubscriptionKit is a *lossless conveyor* of non-final values. It conveys all of 
+an async iteration’s non-final values, as well as the termination (if not necessarily the 
+actual final value).
```

Comment:
Both the subscription kit and the notifier kit *do* convey the final values. In either case, Alice can't see them because she chose to use a `for-await-of` loop to consume them. This is a limitation of JavaScript's `for-await-of` loop regarding *any* asynchronous iteration, and has nothing to do with any limitation of either of our kits.